app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -649,7 +653,10 @@ function useGrantDetail() {
     form.value.payee = grant.value?.payee || '';
     form.value.name = grantMetadata.value?.name || '';
     form.value.description = grantMetadata.value?.description || '';
-    form.value.logoURI = ipfs.getMetaPtr({ cid: grantMetadata.value?.logoPtr?.pointer }) || undefined;
+    form.value.logoURI =
+      ipfs.getMetaPtr({ cid: grantMetadata.value?.logoPtr?.pointer }) !== 'https://scopelift.b-cdn.net/ipfs/'
+        ? ipfs.getMetaPtr({ cid: grantMetadata.value?.logoPtr?.pointer })
+        : '/placeholder_grant.svg';
```

Comment:
Instead of hardcoding `https://scopelift.b-cdn.net/ipfs/` - could we just check if `grantMetadata.value?.logoPtr?.pointer` holds a value:

```suggestion

    const cid = grantMetadata.value?.logoPtr?.pointer;
    form.value.logoURI = cid ? ipfs.getMetaPtr({ cid }) : '/placeholder_grant.svg';
```

Reply:
yeah that's better